### PR TITLE
Update docker-compose release notes for 1.25.5

### PR DESCRIPTION
**Pull request by ulyssessouza**
Bump docker-compose to 1.25.5

### CODE REVIEW

**Comment by ulyssessouza** [comment]
@usha-mandya PTAL

File: compose/release-notes.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ redirect_from:
   - /release-notes/docker-compose/
 ---
 
+## 1.25.5
+(2020-04-10)
+
+### Features
+
+- Bump OpenSSL from 1.1.1d to 1.1.1f
```
Comment (usha-mandya) [inline compose/release-notes.md:15]: Bumped OpenSSL..?

File: compose/release-notes.md [hunk 2]
```diff
@@ -7,6 +7,16 @@ redirect_from:
...
+
```
Comment (usha-mandya) [inline compose/release-notes.md:16]: Could you remove the extra line?

File: compose/release-notes.md [hunk 3]
```diff
@@ -7,6 +7,16 @@ redirect_from:
...
+- Add 3.8 compose version
```
Comment (usha-mandya) [inline compose/release-notes.md:17]: Added Compose version 3.8.

File: compose/release-notes.md [hunk 4]
```diff
@@ -7,6 +7,16 @@ redirect_from:
...
+Limit service scale to the size specified by the field "deploy.placement.max_replicas_per_node"
```
Comment (usha-mandya) [inline compose/release-notes.md:18]: Missing the list item format `-`.
Limited service scale to the size specified by the field `deploy.placement.max_replicas_per_node`.

**Comment by usha-mandya** [comment]
Thanks @ulyssessouza. Added some minor comments. Looks good otherwise.

File: compose/release-notes.md (hunk 4 above)
Comment (ulyssessouza) [reply compose/release-notes.md:18]: Yep. Actually I would like to put this under the "- Add 3.8 compose version" like a sub-item

File: compose/release-notes.md (hunk 2 above)
Comment (ulyssessouza) [reply compose/release-notes.md:16]: I don't see any extra line. Line 16 is just to separate the 2 features

File: compose/release-notes.md (hunk 4 above)
Comment (usha-mandya) [reply compose/release-notes.md:18]: - Added Compose version 3.8.

  - Limited service scale to the size specified by the field `deploy.placement.max_replicas_per_node`.

Comment (usha-mandya) [reply compose/release-notes.md:18]: This adds a sub-item under the Compose version entry.

File: compose/release-notes.md (hunk 2 above)
Comment (usha-mandya) [reply compose/release-notes.md:16]: Yea, an extra line isn't required between bulleted list items. We can ignore this for now as most of the entries within the Compose release notes have an extra line to separate list items.

File: compose/release-notes.md (hunk 4 above)
Comment (ulyssessouza) [reply compose/release-notes.md]: Done!

File: compose/release-notes.md (hunk 3 above)
Comment (ulyssessouza) [reply compose/release-notes.md]: Done

**Comment by ulyssessouza** [comment]
Thanks for the review! PTAL

File: compose/release-notes.md (hunk 1 above)
Comment (ulyssessouza) [reply compose/release-notes.md]: Done

**Pull request merged**